packages/gitbook-v2/src/lib/env/globals.ts

```diff
@@ -64,6 +64,11 @@ export const GITBOOK_DISABLE_TRACKING = Boolean(
 export const GITBOOK_INTEGRATIONS_HOST =
     process.env.GITBOOK_INTEGRATIONS_HOST || 'integrations.gitbook.com';
 
+/**
+ * Hostname for fonts.
+ */
+export const GITBOOK_FONTS_URL = process.env.GITBOOK_FONTS_URL || 'https://fonts.gitbook.com';
```

Comment:
You need to update the `.github/workflows` to pass this env; and also add the value to the vault in 1Password. (So that staging can use the proper value)